src/main/kotlin/uk/gov/justice/digital/hmpps/templatepackagename/config/WebClientConfiguration.kt

```diff
@@ -9,9 +9,9 @@ import java.time.Duration
 
 @Configuration
 class WebClientConfiguration(
-  @Value("\${hmpps-auth.url}") val hmppsAuthBaseUri: String,
-  @Value("\${api.health-timeout:2s}") val healthTimeout: Duration,
-  @Value("\${api.timeout:20s}") val timeout: Duration,
+  @param:Value("\${hmpps-auth.url}") val hmppsAuthBaseUri: String,
```

Comment:
these only ever need to be applied to the parameter so added `param` prefix here